src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTests.cs

```diff
@@ -1,7 +1,10 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Security.Cryptography.Pkcs;
 using Microsoft.AspNetCore.Http.Generators.StaticRouteHandlerModel;
+using Microsoft.Diagnostics.Runtime.ICorDebug;
+using Newtonsoft.Json.Linq;
```

Comment:
These new usings look suspect

Reply:
Not even sure how they got in there! Removing.